internal/cmd/database/create.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/planetscale/cli/internal/cmdutil"
 	"github.com/planetscale/cli/internal/printer"
 
+	"github.com/planetscale/planetscale-go/planetscale"
```

Comment:
You duped the import here too. :)

Reply:
🤦🏾

Reply:
Fixed (again) in https://github.com/planetscale/cli/pull/306/commits/23ead1fbeabfcc71820e043f8cd058146e9970e5